extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -262,97 +483,215 @@ The vendor service configuration must reference the service ID and can contain a
+
+If publisher wishes to ignore a score factor they may either explicitly set its value to `0` or omit it from the `score` map.
+
+Available scoring factors:
+
+1. `supportsViewer` returns `1` when a service can cooperate with the current AMP viewer environment for this page view.
+1. `isReadyToPay` returns `1` when the user is known to the service and the service has a form of payment on file allowing a purchase without entering payment details.
+
+All scoring factors have default value of `0`. In the event of a tie the local service wins.
+
+**Note:** If you would like to test the behavior of a document in the context of a particular viewer, you can add `#viewerUrl=` fragment parameter. For instance, `#viewerUrl=https://www.google.com` would emulate the behavior of a document inside a Google viewer.
+
+### Fallback entitlement
+
+If all configured services fail to get the entitlements, the entitlement configured under `fallbackEntitlement` section will be used as a fallback entitlement for `local` service. The document is unblocked based on this fallback entitlement.
+
+Example fallback entitlement:
 ```js
 {
-  "granted": true/false,
-  "grantReason": "SUBSCRIBER/METERING",
-  "data" : {...}
+  "fallbackEntitlement": {
+    "source": "fallback",
+    "granted": true,
+    "grantReason": "SUBSCRIBER",
+    "data": {
+      "isLoggedIn": false
+    }
+  }
 }
 ```
 
-The properties in the Entitlement response are:
+### Structured data markup
 
-- `granted` - boolean stating if the access to the document is granted or not.
+`amp-subscriptions` relies on the Schema.org page-level configuration for two main properties:
 
-- `grantReason` - the string of the reason for giving the access to the document, recognized reasons are either SUBSCRIBER meaning the user is fully subscribed or METERING meaning user is on metering.
+1.  The product ID that the user must be granted to view the content.
+2.  Whether this content requires this product at this time.
 
-- `data` - free-form data which can be used for template rendering, e.g. messaging related to metering or article count.
+The JSON-LD and Microdata formats are supported.
 
-In cases where the access is granted by a means other than the Entitlement response, messaging via the data property may not be seen by the user. Do not use `data` for granting/denying access to content, conditional display of content based on user access, or displaying user or account related information.
+More detail on the markup is available [here](https://developers.google.com/search/docs/data-types/paywalled-content).
 
-Notice, while it's not explicitly visible, all vendor services also implement authorization endpoints of their own and conform to the same response format.
+#### JSON-LD markup
 
-## Pingback endpoint
+Using JSON-LD, the markup would look like:
```

Comment:
Will it always look like this, or is this an example?

Reply:
Updated with "Below is an example of...:"